### PR TITLE
[OrderByComparator] LPD-26935

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPD-26935](https://liferay.atlassian.net/browse/LPD-26935)

[LPD-26935]: https://liferay.atlassian.net/browse/LPD-26935?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:report:45758236

**Comment by Valmir-unicap** [comment]
Regression found into test UpgradeCatchAllException

- Failed: Unable to process pattern new `FolderIdComparator()` or there is no test associated with it.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -152,6 +156,38 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		FolderArticleModifiedDateComparator.getInstance(false);
 		FolderArticleTitleComparator.getInstance(false);
 		FolderIdComparator.getInstance(false);
```
Comment (Valmir-unicap) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:158]: There is already a test case for `FolderIdComparator`

**Review by Valmir-unicap — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 2]
```diff
@@ -152,6 +156,38 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
...
+		FolderIdComparator.getInstance(false);
```
Comment (Valmir-unicap) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:159]: There is already a test case for `FolderIdComparator`

**Review by Valmir-unicap — CHANGES_REQUESTED** [review]

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava (hunk 2 above)
Comment (victorhcunha) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava]: Thanks @Valmir-unicap! Another class with the same name was merged, so the `FolderIdComparator` became duplicated. I fixed it.

**Comment by Valmir-unicap** [comment]
No regression found!

**Comment by victorhcunha** [comment]
Pushing only to fix commit message

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
Pushing to solve conflicts

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:forward:force

**Comment by victorhcunha** [comment]
ci:report:55402488

**Comment by victorhcunha** [comment]
ci:report:45758236

**Comment by victorhcunha** [comment]
Thanks @dantewang !

**Comment by victorhcunha** [comment]
ci:forward:force

**Pull request closed without merging**